packages/builder/src/types.ts

```diff
@@ -44,6 +44,14 @@ export type ContractData = {
   highlight?: boolean;
 };
 
+export type ArtifactContracts = {
```

Comment:
something like

`export type ArtifactContracts = { artifacts: Pick<ChainArtifacts, 'contracts'> };`

should work

also, is the `artifacts` wrapper really necessary?